includes/class-ajax-metabox.php

```diff
@@ -0,0 +1,571 @@
+<?php
+/**
+ * Optimizely X: AJAX_Metabox class
+ *
+ * @package Optimizely_X
+ * @since 1.0.0
+ */
+
+namespace Optimizely_X;
+
+/**
+ * Defines AJAX endpoints that communicate with
+ * the API via the Optimizely X Metabox
+ *
+ * @since 1.0.0
+ */
+class AJAX_Metabox extends AJAX {
+
+	use Singleton;
+
+	/**
+	 * An AJAX endpoint for optimizely_x_change_status.
+	 *
+	 * @since 1.0.0
+	 * @access public
+	 */
+	public function change_status() {
```

Comment:
This function doesn't seem to be doing a permissions check via current_user_can() or something similar.